lib/ProductOpener/Images.pm

```diff
@@ -541,47 +576,9 @@ sub process_image_upload($$$$$$) {
 			}
 
 			$source->Set('quality',95);
-			$x = $source->Write("jpeg:$www_root/images/products/$path/$imgid.jpg");
-
-			# Check that we don't already have the image
```

Comment:
We still need to keep the file size check so that we check collisions with the existing files that have no hash in the filename. That's in particular useful for imports of data from other databases, producers etc. It may not be efficient, but currently we import again existing images and they are discarded when uploaded.